Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -614,6 +615,14 @@ public void dispose() {
     return myServerData.getErrors(file);
   }
 
+  public List<DartServerData.DartError> getErrors(@NotNull final Module module) {
```

Comment:
Don't you mind me removing `getErrors(Module)` here and in `DartServerData`? You always can use `module.getModuleContentScope()` instead.